doc/nrf/protocols/wifi/sniffer_rx_operation.rst

```diff
@@ -0,0 +1,136 @@
+﻿.. _ug_nrf70_developing_raw_ieee_80211_packet_reception:
+
+Raw IEEE 802.11 packet reception using Monitor mode
+###################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF70 Series device supports Monitor mode, that is, listening for and receiving all IEEE 802.11 traffic on a given channel, not restricted to a certain SSID. With Monitor mode raw IEEE 802.11 packets can be received directly at the user application layer.
+The nRF70 device has to be configured in monitor mode in order to operate as an 802.11 wireless packet sniffer.
+
+In Monitor mode, raw 802.11 packets received at the nRF70 Series device may be presented directly at the host application layer. When forwarding the received IEEE 802.11 packets to the user application, the Wi-Fi host driver will append a raw RX header to each received packet and deliver it up to the application layer using the raw socket API.
+The raw RX header holds the receive parameters for the received packet, such as data rate, receive signal strength, frequency of reception, and packet type (whether it is Legacy, HT, VHT, or HE).
+
+.. _ug_nrf70_developing_enabling_802.11_monitor_mode:
+
+Enabling Monitor mode
+*********************
+
+To be able to use the Monitor mode in your application, you need to enable the :kconfig:option:`CONFIG_NRF700X_RAW_DATA_RX` Kconfig option in the project configuration.
+
+.. _ug_nrf70_developing_monitor_mode_operation:
+
+Monitor mode operation
+**********************
+
+To enable the Monitor mode in the nRF Wi-Fi driver at runtime, use the ``NET_REQUEST_WIFI_MODE`` network management runtime API.
+This runtime API can be used to disable the Monitor mode in the driver, when packet sniffer operation is not required anymore in the user application.
+
+When the nRF70 device is configured to operate in Monitor mode, simultaneous connected Station operation is not permitted (the nRF70 Series device operates exclusively as a Wi-Fi sniffer).
+
+The user needs to configure the operating Wi-Fi channel on which the nRF70 device will monitor and receive 802.11 packets.
```

Comment:
It is mandatory to configure an operating channel on the nRF70 device before monitor mode can be enabled on it. This can done using ``NET_REQUEST_WIFI_CHANNEL`` runtime NET-management API.

Reply:
The API is discussed below. 
"mandatory" is probably implied by "needs to"

Reply:
Yes, that is correct.